MapboxVision/VisionManager/VisionManagerProtocol.swift

```diff
@@ -14,15 +14,17 @@ public protocol VisionManagerProtocol: AnyObject {
      Converts location of the point from a screen coordinate to a world coordinate.
 
      - Parameter screenCoordinate: Screen coordinate expressed in pixels
+     - Returns: World coordinate if `screenCoordinate` can be projected on the road and nil otherwise
      */
-    func pixelToWorld(screenCoordinate: Point2D) -> WorldCoordinate
+    func pixelToWorld(screenCoordinate: Point2D) -> WorldCoordinate?
 
     /**
      Converts location of the point from a world coordinate to a screen coordinate.
 
      - Parameter worldCoordinate: Point in world coordinate
+     - Returns: Screen coordinate if `worldCoordinate` can be represented in screen coordinates
```

Comment:
Is "... and nil otherwise" missing here?

Reply:
fixed